book/src/ch02-00-guessing-game.md

```
@@ -0,0 +1,215 @@
+    - Random number generation
+    - String formatting
+    - Input handling
+- Loops
+- Variables
+
+These topics will all be covered far more thoroughly in later chapters, but this classic game is
+a perfect opportunity to build something tangible for the first time.
+
+As with all projects, start by creating a project directory.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)